src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    private Bulldozer model = new Bulldozer();
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> listBulldozer = new ArrayList<>();
```

Comment:
```suggestion
        List<Bulldozer> bulldozers = new ArrayList<>();
```
don't need to add the name of the class or interface as part of the variable name in such cases